docs/getting_started/supported_tasks.rst

```diff
@@ -24,7 +24,7 @@ Rubrix is flexible with input and output shapes, which means you can model many
 * `Key phrase extraction <https://paperswithcode.com/task/keyword-extraction>`_
```

Comment:
Very good job @ignacioct . I'd remove this from here, I think I added this accidentally.

Reply:
Done!